SECURITY.md

```diff
@@ -0,0 +1,19 @@
+# Security Policy
+
+## Supported Versions
+
+Use this section to tell people about which versions of your project are
+currently being supported with security updates.
+
+| Version | Supported          |
+| ------- | ------------------ |
+| 1.2.x   | :x:                |
+| 1.3.x   | :white_check_mark: |
+
+## Reporting a Vulnerability
+
+Please report security concerns to security@chia.net.
+
+If your security issue is established to be valid, we will reach out immediately to establish
+communication channels and compensate the issue reporter for responsibly reporting security bugs via
+our bug bounty program. 
```

Comment:
Might be nice to link to info on the BBP? Like https://www.chia.net/2021/10/21/bugcrowd-bounty-launch.en.html

Reply:
we cannot link to bug bounty now because its not open to the public. This is an ongoing thing @wriches is sorting out with them.